etc/checkstyle-suppressions.xml

```diff
@@ -25,5 +25,10 @@
     <suppress checks="FileLength"
             files="config/config/src/main/java/io/helidon/config/Config.java"
             lines="1"/>
+
+        <!-- Java comments with AsciiDoc tag:: and end:: in import section incorrectly flagged
```

Comment:
I can foresee the list of files growing.
How about using the comments to disable checkstyle.

```java
//CHECKSTYLE:OFF
xyz
//CHECKSTYLE:ON
```
See http://checkstyle.sourceforge.net/config_filters.html#SuppressionCommentFilter

Reply:
I looked at that (very) briefly but did not try it. In my admittedly quick reading about it it seemed that using the suppression comment filter required more configuration and uses a module which I _thought_ required a later version of `check style` than the one we are using, but I could be wrong. Of course we could think about going to the later version if needed but to start with I wanted to minimize what I needed to do to work around this.